.github/workflows/package-publish.yml

```diff
@@ -27,6 +27,33 @@ jobs:
         with:
           node-version: 18.x
           cache: 'yarn'
+      - name: 'validation'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const permission = await github.rest.repos.getCollaboratorPermissionLevel({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              username: context.actor
+            });
+            if (permission.data.permission !== 'admin' && permission.data.permission !== 'write') {
+              core.setFailed(`User ${context.actor} does not have write or admin permissions to this repository.`);
+            }
+            const currentBranch = context.ref.replace('refs/heads/', '');
+            if (currentBranch !== `release/v${{ github.event.inputs.version }}`) {
+              core.setFailed(`Current branch ${currentBranch} does not match release/v${{ github.event.inputs.version }}`);
+            }
+            const pullRequests = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open',
+              sort: 'created',
+              direction: 'desc',
+            });
+            const pr = pullRequests.data.find(pr => pr.head.ref === currentBranch && pr.base.ref === 'main');
```

Comment:
is it main or develop?

Reply:
Thank you for pointing that out. It should be `develop` instead of `main`